README.md

```diff
@@ -37,20 +39,25 @@ These principles are what make Amethyst unique and competitive in the world of g
 
 ### Extreme Multithreading
 
-Amethyst is based over a very powerful parallel [ECS][ecs] called Specs.
-This allows games built with Amethyst to maximize the processing power usage to make it run as smooth and as fast as possible, without the headache of multithread programming.
+Amethyst is based over a very powerful parallel [ECS][ecs] called Specs. This
+allows games built with Amethyst to maximize the available processing power to
+run as smoothly and as quickly as possible, without the headaches of
+multi-threaded programming.
```

Comment:
Those line changes do not make any visual differences. Are they there so it is easier to read in a text editor?

Reply:
Oh, that must have been because of my auto-formatting for Markdown documents in my text editor. I can revert them, if you'd like!